docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+
+XHsZ943WBg5RPNhVAT8x
+```
+
+Let’s now run the uploaded cql script and check that you can read the data (don’t forget to replace the password with the one extracted in the previous step):
+
+```
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -f /tmp/test_data.cql
+
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "SELECT * FROM medusa_test.users"
+
+ email             | name          | state
+-------------------+---------------+-------
+ alice@example.com |   Alice Smith |    TX
+   bob@example.com |     Bob Jones |    VA
+ david@example.com |    David Yang |    NV
+ carol@example.com | Carol Jackson |    CA
+
+(4 rows)
+```
+
+Now backup this data, and check that files get created in your MinIO bucket. 
+
+To that end, use the following command:
+
+```
+helm install my-backup k8ssandra/backup -n k8ssandra --set name=backup1,cassandraDatacenter.name=dc1
+```
+
+Since the backup operation is asynchronous, you can monitor its completion by running the following command:
+
+```
+kubectl get cassandrabackup backup1 -n k8ssandra -o jsonpath={.status.finishTime}
+```
+
+As long as this doesn’t output a date and time, then the backup is still running. With the amount of data present and the fact that you’re using a locally accessible backend, this should complete quickly.
+
+Now refresh the MinIO UI and you should see some files in the k8ssandra-medusa bucket:
+
+![K8ssandra Backup Files](k8ssandra-medusa-backup.png)
+
+An index folder should appear (it is Medusa’s backup index) and then another folder that is specific to each Cassandra node in the cluster (in this case there is only one node).
+
+# Deleting the data and restoring the backup
+
+Now delete the data by truncating the table, and check that the table is empty:
+
+```
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "TRUNCATE medusa_test.users"
+
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "SELECT * FROM medusa_test.users"
+
+ email | name | state
+-------+------+-------
+
+(0 rows)
+```
+
+Now restore the backup taken previously:
+
+```
+helm install restore-test k8ssandra/restore --set name=restore-backup1,backup.name=backup1,cassandraDatacenter.name=dc1 -n k8ssandra
+```
+
+This operation will take a little longer as it requires to stop the StatefulSet pod and perform the restore as part of the init containers, before the Cassandra container can start. You can monitor progress using this command:
+
+```
+watch -d kubectl get cassandrarestore restore-backup1 -o jsonpath={.status} -n k8ssandra
+```
+
+The restore operation is fully completed once the **finishTime** value appears in the output:
```

Comment:
Same question as above re: condition

Reply:
I think it's the same as backups.

Reply:
Resolving based on @jsanda's response above